packages/mdc-grid-list/mdc-grid-list.scss

```diff
@@ -0,0 +1,237 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+
+$mdc-grid-list-tile-width: 200px;
```

Comment:
This should be made `!default` so people can alter the size of grid tiles, or should it always be the same size. Furthermore, both this property as well as the `mdc-grid-tile-width` should be documented in the README.